validate.py

```diff
@@ -157,9 +140,8 @@ def email_results(subject, results_text_string):
     if len(flags) == 0:
         flags.append("GREEN")
     flag_prefix = f"[{', '.join(flags)}] "
-    email_results(flag_prefix + job, results_text)
+    now = datetime.now(tz=pytz.UTC)
+    print(f"{flag_prefix}{job} for {now:%B %d, %Y}\n\n{results_text}")
```

Comment:
May want to add a space between `flag_prefix` and `job`?

Reply:
When the space was in there it had two spaces in the output. This is what it looks like
[GREEN] UDW Daily Status Report for October 24, 2019. Probably because the flag_prefix line before this has a space afterward already.

Reply:
Oh, sorry, missed that! Yeah, probably removing the space from the end of the `flag_prefix` assignment is best, but it's not a big deal either way.

Reply:
Yeah I swapped them thx